packages/components/accordion/src/use-accordion-item.ts

```diff
@@ -0,0 +1,138 @@
+import type { PropGetter } from "@yamada-ui/core"
+import type { KeyboardEvent, KeyboardEventHandler } from "react"
+import type { AccordionItemOptions } from "./accordion-item"
+import { ariaAttr, handlerAll, isArray, mergeRefs } from "@yamada-ui/utils"
+import { useCallback, useId } from "react"
+import {
+  useAccordionContext,
+  useAccordionDescendant,
+} from "./accordion-context"
+
+interface useAccorDionItemProps
```

Comment:
@kaehehehe 
I would like the type definitions to be written in PascalCase.Can you change it to `UseAccorDionItemProps`?

Reply:
Fixed!